docs: refactor Getting Started landing page DOC-1067 DOC-1069 (#2345)
spectrocloud/librarium: docs/docs-content/getting-started/getting-started.md

sidebar_label: "Getting Started"
title: "Getting Started"
description: "Spectro Cloud Getting Started"
hide_table_of_contents: true
sidebar_custom_props:
  icon: "overview"
tags: ["getting-started"]
---

This page gives you an overview of how to get started with Spectro Cloud Palette and begin leveraging its Kubernetes
full-stack management at scale. Palette's unique capabilities provide end-to-end declarative cluster management, cluster
monitoring and reconciliation, as well as enterprise-grade security.

> Check out the following short video for an overview of some the key functionality that Palette provides. Spectro Cloud
> Palette provides developers and platform engineers with complete control of their Kubernetes clusters. It provides
> support for public cloud providers, data centers, bare metal and edge, so you can use Palette no matter the complexity
> or location of your production environments.

Reviewer: ```suggestion
Palette provides developers and platform engineers with complete control of their Kubernetes clusters. The platform provides support for public cloud providers, data centers, bare metal, and edge so that you can deploy and use Kubernetes no matter the complexity or location of your production environments. Check out the following short video for a quick overview of Palette's key functionalities.
```

Changed the order so that when we state "Check out the following video" the video is actually the next item :)

<YouTube
  url="https://www.youtube.com/embed/P9QBOifS_cc"
  title="Demo | Spectro Cloud Palette - Kubernetes-as-a-Service"
/>

<br />

## Deploy Kubernetes Clusters with Palette

The first step towards adopting Palette in your organization is to
[create a login](https://www.spectrocloud.com/get-started). The
[Palette Free Tier](https://www.spectrocloud.com/free-tier) allows you to experience the benefits of Palette first-hand.

We have curated the pages in the Getting Started section to give you a gradual introduction to the fundamental concepts

> and workflows you need to deploy Kubernetes clusters to Palette.

Reviewer: ```suggestion
and workflows you need to deploy and manage Kubernetes clusters through Palette.
```
Changed it to "through Palette" as we don't want the reader to think we are compute platform hosting the k8s cluster :)

<div class="desktop-only-display">

![Overview of the getting started journey rocket](/getting-started/getting-started_getting-started_journey-overview.png)

</div>

<br />

> First, you learn how to create cluster profiles, which are the blueprints to your Kubernetes stacks. Then, you deploy
> your cluster profiles with Palette. Once you have deployed your first cluster, you update it using Palette's cluster
> management functionality. The Getting Started section also includes other topics that you can explore further such as
> cluster deployment with Terraform, Palette's edge capabilities and virtual machine orchestration.

Reviewer: ```suggestion
First, you will learn how to create cluster profiles, which are the blueprints for your Kubernetes stacks. Then, you will deploy
your first cluster using your cluster profile. Once you have deployed your first cluster, you update it using Palette's cluster
management functionality. The Getting Started section also includes other topics that you can explore further, such as
cluster deployment with Terraform, Palette's edge capabilities, and virtual machine orchestration.
```
Minor language touches
